src/SCRIPTS/BF/pages.lua

```diff
@@ -4,6 +4,10 @@ if apiVersion >= 1.036 then
     PageFiles[#PageFiles + 1] = { title = "VTX Settings", script = "vtx.lua" }
 end
 
+if apiVersion >= 1.016 then
+    PageFiles[#PageFiles + 1] = { title = "Profiles", script = "profiles.lua" }
```

Comment:
Move to line 12?

Reply:
By having it like this the pages can be shuffled around as we please. and if a page is to be removed we have to break it out anyway. I also like the readability of the current approach of having a if for each page.